examples/gkurve/LICENSE.tesselator

```diff
@@ -0,0 +1 @@
+TODO: add license, the tesselator implementation comes from https://github.com/fubark/cosmic/blob/master/graphics/src/tessellator.zig
```

Comment:
@fubark We're using your Cosmic tesselator implementation, and plan to continue doing so in the future. For now @PiergiorgioZagaria has extracted it from Cosmic so we can use it here, I'm curious how you'd feel about having this live in a separate repository/library proper?

(if that's just more work for you, no worries, just figured I'd ask!)